zhaquirks/xiaomi/aqara/motion_agl1.py

```diff
@@ -0,0 +1,192 @@
+"""Quirk for aqara lumi.sensor_occupy.agl1."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from zigpy import types
+from zigpy.quirks.v2 import (
+    NumberDeviceClass,
+    QuirkBuilder,
+    SensorDeviceClass,
+    SensorStateClass,
+)
+from zigpy.quirks.v2.homeassistant import EntityType, UnitOfLength
+from zigpy.zcl.clusters.general import DeviceTemperature
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+from zhaquirks import LocalDataCluster
+from zhaquirks.xiaomi import XiaomiAqaraE1Cluster
+
+
+class AqaraMotion(types.enum8):
+    """Aqara motion attribute values."""
+
+    Idle = 0x02
+    Moving = 0x03
+    Still = 0x04
+
+
+class AqaraMotionSensitivity(types.enum8):
+    """Aqara motion sensitivity attribute values."""
+
+    Low = 0x01
+    Medium = 0x02
+    High = 0x03
+
+
+class AqaraOccupancy(types.enum8):
+    """Aqara occupancy attribute values."""
+
+    Unoccupied = 0x00
+    Occupied = 0x01
+
+
+class IasZoneLocal(LocalDataCluster, IasZone):
+    """Virtual cluster for IasZone."""
+
+    _CONSTANT_ATTRIBUTES = {
+        IasZone.AttributeDefs.zone_type.id: IasZone.ZoneType.Motion_Sensor
+    }
+    _VALID_ATTRIBUTES = {IasZone.AttributeDefs.zone_status.id}
```

Comment:
Note for other reviewers: We need a virtual cluster for `IasZone` here, as we need a virtual attribute for "motion". The Xiaomi `motion` attribute needs to be parsed in a way that we can't yet do with quirks v2.

Also, for motion/occupancy, it makes sense to use the integrated entities in ZHA IMO. We'd have to duplicate "motion" and "occupancy" quirk entities with all properties everywhere otherwise.

We already use virtual clusters in a lot of quirks and they'll always be needed for some (especially for Tuya and some Xiaomi devices), unless we have some kind of other structure we can map attributes to (like Z2M does).